.circleci/config.yml

```diff
@@ -342,6 +342,7 @@ jobs:
       - NODE_VERSION: << pipeline.parameters.node-version >>
     steps:
       - build-win
+      - package-extensions
```

Comment:
See https://github.com/forcedotcom/salesforcedx-vscode/pull/4510/files#r1003744311 about why this is necessary.